### ISSUE TITLE
Common "jar" for those wallets that sync the same blockchain (optimization)

### ISSUE THREAD

**Issue by serrq** [horizontalsystems/unstoppable-wallet-android#6635]
Currently I have three wallets, that I called (fantasy names) Bitkipi, Pluto and Saturn.
Made exception for BitKipi that has just a BIP84 Bitcoin wallet, Pluto and Saturn are identical (as blockchains I mean).

My main wallet is Saturn. 
I launch Saturn several times at day, so it is always properly synced (if there aren't issues).

Why I have to run (download) multiple syncs for the same blockchain in different wallets?

When Saturn is synced, Pluto is not if I launch it after several days.

Why don’t we use a common synchronization jar between all the wallets if they have common blockchains?

In conclusion the concept is:

Same jar (synced blockchain), many synced wallets.
